src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -500,7 +493,7 @@ namespace OpenLoco::ObjectManager
         entry._name = (char*)*ptr;
         *ptr += strlen(entry._name) + 1;
 
-        // ObjectHeader3* h3 = (ObjectHeader3*)ptr;
+        entry._displayData = reinterpret_cast<ObjectHeader3*>(ptr);
```

Comment:
There's a mistake here:
```suggestion
        entry._displayData = reinterpret_cast<ObjectHeader3*>(*ptr);
```